desc/compute/_equil.py

```diff
@@ -802,3 +803,37 @@ def _beta_volpol(params, transforms, profiles, data, **kwargs):
 def _beta_voltor(params, transforms, profiles, data, **kwargs):
     data["<beta_tor>_vol"] = jnp.abs(data["W_p"] / data["W_Btor"])
     return data
+
+
+@register_compute_fun(
+    name="P_ISS04",
+    label="P_{ISS04}",
+    units="W",
+    units_long="Watts",
+    description="Heating power required by the ISS04 energy confinement time scaling",
+    dim=0,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="",
+    data=["a", "iota", "rho", "R0", "W_p", "<ne>_rho", "<|B|>_axis"],
+    method="str: Interpolation method. Default 'cubic'.",
+    H_ISS04="float: ISS04 confinement enhancement factor. Default 1.",
+)
+def _P_ISS04(params, transforms, profiles, data, **kwargs):
+    rho = transforms["grid"].compress(data["rho"], surface_label="rho")
+    iota = transforms["grid"].compress(data["iota"], surface_label="rho")
+    iota_23 = interp1d(2 / 3, rho, iota, method=kwargs.get("method", "cubic"))
```

Comment:
If your objective already computes `iota_r`, you can do this to give more accurate result
```python
fx = {}
if "iota_r" in data:
  fx["fx"] = transforms["grid"].compress(data["iota_r"])  # noqa: unused dependency
iota_23 = interp1d(2 / 3, rho, iota, method=kwargs.get("method", "cubic"), **fx)
```